// Wrong number of arguments
function test(a: number, b: string): void {
    console.log(a, b)
}

test(42)
